src/components/Books.js

```diff
@@ -0,0 +1,64 @@
+function Book() {
+  return (
+    <div className="container">
+      <div className="infocard">
+        <span>Action</span>
+        <h1>The hunger games</h1>
+        <p>Suzanne Collins</p>
+        <ul>
+          <li>Comments |</li>
+          <li>Remove |</li>
+          <li>Edit</li>
```

Comment:
- [ ] According to the project requirements. You should have :
 * 1. `BookList` component that will:
    - Renders a list of books using the `Book` component and 
    - Renders a form by using the CreateNewBook component
* 2. `Book` component that should:
  * Takes the book data as a prop and renders the title and the author
  * Has a button for removing books
- 3. CreateNewBook component ( You can use another name eg: `Form`) I can see you have it already. 👍🏻 

- [ ] Kindly make sure to address this. ✍🏻 
 _ You can check this [repo](https://github.com/anasbassi/BookStore/pull/1) for more if you have any question